pennylane/resource/resource.py

```diff
@@ -49,10 +51,13 @@ class Resources:
         shots: 0
         gate_types:
         {'Hadamard': 1, 'CNOT': 1}
+        gate_sizes:
+        {1: 1, 2: 1}
```

Comment:
Non-blocking comment: I sometimes find this field a bit confusing and need to think to remember what exactly these numbers are. Is there a way to make it more clear with minimal work? e.g.,  `{'1-qubit': 1, '2-qubit': 1}`

Reply:
I think this is a great suggestion, probably need a separate PR as this would be a breaking change for users who expect it to be an int.